### DISCUSSION TITLE
What's seed? and how too change the type of emotion with out it saying so?

### DISCUSSION THREAD

**Discussion by RayJay232** [JarodMica/ai-voice-cloning#99] [Q&A]
hi i am new to this and i have no clue where to find information on seed.... so anything that can help would be great?    also from my point of view/guess is that it changes the tone?

**Comment by NathanInArk** [comment]
I'm a bit of a noob myself, but as far as I understand it, a seed acts as a starting point. It's often picked via a random number generator. So by selecting a seed you are helping influence the output in a predictable way by giving it the same starting point every time.

**Comment by mribah78** [comment]
I am not completely sure, but in Stable Diffusion (AI image generation) the "seed" is a large random number that is the starting point for the data that is then refined by the model to produce the end result.  If you have the same seed, and leave all of the settings and inputs the same, you will get the exact same result every time.  By default the seed is set to random for each generation attempt, meaning you will get slightly different results each time.  Your inputs (the text input, the iterations, samples, temperature, etc.) "steer" the seed towards what you want, so even with a random seed you'll get similar results but not duplicate.  If your directions/inputs are the same with the same seed, the algorithms will do the exact same thing and you'll get the exact same result.
